content/references/rippled-api/public-rippled-methods/subscription-methods/subscribe.md

```diff
@@ -534,5 +535,24 @@ The format of an order book stream message is the same as that of [transaction s
 | `transaction.owner_funds` | String | Numeric amount of the `TakerGets` currency that the `Account` sending this OfferCreate transaction has after executing this transaction. This does not check whether the currency amount is [frozen](freezes.html). |
 
 
+## Consensus Stream
+
+The `consensus` stream sends `consensusPhase` messages when [the consensus process](consensus.html) changes phase. The message contains the new phase of consensus the server is in.
+
+```
+{
+  "type": "consensusPhase",
+  "consensus": "accepted"
+}
+```
+
+The fields from a consensus stream message are as follows:
+
+| `Field`             | Type                      | Description                |
+|:--------------------|:--------------------------|:---------------------------|
+| `type`              | String                    | `consensusPhase` indicates this is from the consensus stream |
+| `consensus`         | String                    | The new consensus phase the server is in, one of 'open', 'establish', or 'accepted'.|
```

Comment:
Please change the description to the following:

> The new consensus phase the server is in. Possible values are `open`, `establish`, and `accepted`.

And please make sure there is a space before the closing `|`.

Also, I'd love to have descriptions of how those states correspond to the steps described in the [Consensus article](https://xrpl.org/consensus.html) but we can push this first and add those later.